cmd/util.go

```diff
@@ -344,3 +362,30 @@ func makeSubstitutions(s string, ext string, m map[string]string) string {
 
 	return s
 }
+
+func buildComponentMissingErrorMessage(nodetype check.NodeType, component string, bins []string) error {
```

Comment:
Same comment as before about error messages - the type this returns could be string rather than error